completion/git-machete.completion.zsh

```diff
@@ -213,15 +213,45 @@ __git_machete_directions_show() {
   _describe -t directions 'direction' directions "$@"
 }
 
-__git_machete_github_subcommands() {
-  local github_subcommands
-  github_subcommands=(
-    'anno-prs:annotate the branches based on their corresponding GitHub PR numbers and authors'
-    'checkout-prs:check out the given pull requests locally' # TODO (#393): Add support in zsh and bash completion for options of `git machete github checkout-prs`
-    'create-pr:create a PR for the current branch, using the upstream (parent) branch as the PR base'
-    'retarget-pr:set the base of the current branch PR to upstream (parent) branch'
-  )
-  _describe -t github_subcommands 'subcommand' github_subcommands "$@"
+
+__git_machete_github_subcommands ()
+{
+  local curcontext="$curcontext" state line
+  typeset -A opt_args
+
+  _arguments -C \
+    ':command:->command' \
+    '*::options:->options'
+
+  case $state in
+    (command)
+
+      local -a github_subcommands
+      github_subcommands=(
+        'anno-prs:annotate the branches based on their corresponding GitHub PR numbers and authors'
+        'checkout-prs:check out the given pull requests locally'
+        'create-pr:create a PR for the current branch, using the upstream (parent) branch as the PR base'
+        'retarget-pr:set the base of the current branch PR to upstream (parent) branch'
+      )
+      _describe -t commands 'subcommand' github_subcommands
+      ;;
+
+    (options)
+      case $line[1] in
+
+        (create-pr)
+          _arguments '(--draft)'--draft'[Creates the new PR as draft]'
+        ;;
+
+        (checkout-prs)
+          _arguments \
+            '(--all)'--all'[Checkout all open PRs.]' \
+            '(--by)'--by'[Checkout open PRs authored by the given Github user]' \
+            '(--mine)'--mine'[Checkout open PRs for the current user associated with the Github token.]'
+        ;;
+      esac
+    ;;
+  esac
 }
 
 __git_machete_categories() {
```

Comment:
Okay! if you could add the missing Bash completion... should be much easier than zsh 😅